packages/clay-link/src/ClayLink.soy

```diff
@@ -78,6 +83,9 @@
 			{param label: $label /}
 			{param spritemap: $spritemap /}
 		{/call}
+		{if $title}
```

Comment:
@georgel-pop-lr same here, I think it's not needed. Passing a "title" in `${attributes}` is probably the way to go.

Reply:
I think is needed, the title attribute is not being read by screen readers so if there is no label, the screen readers will have nothing to read. (Using it on most HTML elements to provide information for users of a variety of assistive technologies. Access to title attribute information is not supported uniformly by screen readers https://www.tpgi.com/using-the-html-title-attribute/)
This is an image from the accessibility report:
![image](https://user-images.githubusercontent.com/59687465/143830123-40fab5c0-a975-4853-9544-e3f51fa66904.png)

Reply:
@georgel-pop-lr Ok we saw this together on a Slack call and I know you'll be updating this PR, sorry for the confusion.